src/BloomExe/web/controllers/CollectionSettingsApi.cs

```diff
@@ -266,6 +268,12 @@ public void RegisterWithApiHandler(BloomApiHandler apiHandler)
 			apiHandler.RegisterEndpointHandler(kApiUrlPart + "addCustomPaletteColor", HandleAddCustomColor, false);
 		}
 
+		private void ResetBookshelf()
```

Comment:
I think we also want to use this new method on line 128? (and you can remove the null check there)